workshop/exercise-3.-ibm-cloud-with-ibm-cloudant.md

```diff
@@ -87,6 +178,29 @@ Complete the following tasks:
 
 * [ ] From the Cloudant service Overview page, select **Service credentials** from the left pane, as shown in the following figure.
```

Comment:
This is a comment on a section that did not change

Reply:
Done